.ci.yaml

```diff
@@ -3338,6 +3338,33 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac framework_tests_impeller
```

Comment:
This needs to be x86 due to gold issues, but I commented in https://github.com/flutter/flutter/issues/119880 with an idea to fix that.

Reply:
ohh, i'm sorry, i got mixed up. ahh, so at the top level we have a way of specifying arm-specific, but not intel-specific, huh? This makes sense.